src/main/scala/cz/alenkacz/db/postgresscala/DbValue.scala

```diff
@@ -0,0 +1,6 @@
+package cz.alenkacz.db.postgresscala
+
+class DbValue(value: Any) {
```

Comment:
Maybe this should be a trait and there should be a specific implementation for each underlying driver. The reason is that the raw underlying data types can differ.